src/App.css

```diff
@@ -1,4 +1,5 @@
-.App {
+
+/* .App {
```

Comment:
Do we want to comment out .App styling for the entire application? 
Perhaps you can assign a `className` for the component and apply styling individually to the drawer.

Reply:
Yeah you don't ever need to mess with the .css files of this project because it's just easier to assign a `className` like I had done.